apps/checkout/src/sections/CheckoutForm/useCheckoutFinalize.ts

```diff
@@ -0,0 +1,53 @@
+import { pay as payRequest } from "@/fetch";
+import { useCheckout } from "@/hooks/useCheckout";
+import { useFetch } from "@/hooks/useFetch";
+import { extractMutationErrors } from "@/lib/utils";
+import { useErrors } from "@/providers/ErrorsProvider";
+import { useAuth, useAuthState } from "@saleor/sdk";
+import { FormData } from "./types";
+import omit from "lodash/omit";
+
+export const useCheckoutFinalize = () => {
+  const { checkout } = useCheckout();
+  const { register } = useAuth();
+  const { user } = useAuthState();
+  const [, pay] = useFetch(payRequest, { opts: { skip: true } });
```

Comment:
Wouldn't it make sense to treat second argument as `opts` itself?
```suggestion
  const [, pay] = useFetch(payRequest, { skip: true });
```

Reply:
There is opts and args there, so if we make it opts itself and want to add just the args we'd need to pass empty argument, not feeling great about it but it's not a strong opinion

Reply:
Okay, if we'd assume that, then sure, we could do that 👍